spec/core_functions/color/mix.hrx

```diff
@@ -349,6 +391,16 @@ Error: argument `$weight` of `mix($color1, $color2, $weight: 50%)` must be a num
 a {b: mix(red, blue, -0.001)}
 
 <===> error/bounds/too_low/error
+DEPRECATION WARNING: $weight: Passing a number without unit % (-0.001) is deprecated.
```

Comment:
The deprecation warning was already part of the tests above, is testing both warning + error intentional? If not, then I think it's better to add units so we don't emit both

Reply:
Yeah, good catch